src/BrandEmbassyCodingStandard/Sniffs/Classes/ForbiddenStaticMethodSniff.php

```diff
@@ -90,11 +91,9 @@ private function addMethodCallError(
         int $staticMethodCallPointer,
         string $forbiddenClass
     ): void {
-        $methodNamePointer = $staticMethodCallPointer + 2;
-        $call = TokenHelper::getContent($phpcsFile, $staticMethodCallPointer, $methodNamePointer);
-        $expectedCall = TokenHelper::getContent($phpcsFile, $staticMethodCallPointer + 1, $methodNamePointer);
+        $call = TokenHelper::getContent($phpcsFile, $staticMethodCallPointer, $staticMethodCallPointer);
```

Comment:
Tohle vůbec není call. To je `"self"` nebo `"static"`. Je to statická reference na třídu. Možná self-reference?